lib/pa11y.js

```diff
@@ -31,7 +32,8 @@ module.exports.defaults = {
 		parameters: {
 			'ignore-ssl-errors': 'true',
 			'ssl-protocol': 'tlsv1'
-		}
+		},
+		path: phantomjsPath
```

Comment:
If this doesn't make it in, would Pa11y be willing to allow the path to be configurable via the API? It already is configurable via the CLI.